actpool/config.go

```diff
@@ -13,7 +13,8 @@ var (
 	DefaultConfig = Config{
 		MaxNumActsPerPool:  32000,
 		MaxGasLimitPerPool: 320000000,
-		MaxNumActsPerAcct:  2000,
+		MaxNumActsPerAcct:  500,
```

Comment:
is 500 the current default value used on mainnet nodes?

Reply:
https://github.com/iotexproject/iotex-core/pull/4028/files#diff-a8abac2771195f422f421ee3179639574be3f78960621421a3d5189e8f1bde14L144